app/services/save_practice_service.rb

```diff
@@ -145,6 +145,55 @@ def update_category_practices
     category_params = @practice_params[:category]
     practice_category_practices = @practice.category_practices
     practice_categories = @practice.categories
+    categories_selected = Array.new
```

Comment:
can we move this mailer logic outside of this `update_categories_practices` method to a separate one that just handles this?

Reply:
Yeah - thats a good call Elisa - I'll move it all in to it's own method.

Reply:
Yeah thats cleaner - being in a separate method. thx